### PR TITLE
Modified the firmware documentation to document the new half-duplex feature

### PR DESCRIPTION
**Pull request by prices**
Serial1 can now be put into half-duplex mode.  The feature was added with firmware pull request #461.

### CODE REVIEW

**Comment by m-mcgowan** [comment]
Thank you very much for doing this! Much appreciated. It's rare to see a contribution that includes the code, test code and docs! +:100:

**Comment by prices** [comment]
Next time, I hope to get it complete the first time.  ;)

I must say, that is a pretty slick method of doing documentation.  I love the automatic building and reloading!  I may have to look into that for my work projects.

File: src/content/en/shared/firmware.md [hunk 1]
```diff
@@ -1362,6 +1362,32 @@ Serial1.flush();
...
+// EXAMPLE
+// Initializes Serial1 at 9600 baud and enables half duplex mode
+
+Serial1.begin(9600);
+Serial1.halfduplex(true);
+
+```
+*Parameters:*
+
+- `enable`: true to enable half-duplex mode.  false to disable.
```
Comment (kennethlimcp) [inline src/content/en/shared/firmware.md]: `true`: enables half-duplex mode.
`false`: disable half-duplex mode.

Comment (prices) [inline src/content/en/shared/firmware.md]: Yes.  Should I modify the text to make it more clear?

Comment (kennethlimcp) [inline src/content/en/shared/firmware.md]: another way is to standardize the formatting. using ( ` ` ) for **false** as well.

`enable`: true to enable half-duplex mode.  
`false` to disable.

Comment (prices) [inline src/content/en/shared/firmware.md]: So are you thinking something like:

`halfduplex()` takes one argument:  `true` enables half-duplex mode, `false` disables half-duplex mode.

**Comment by jme783** [comment]
@prices can you pull the new `master` (we updated the docs structure), and resolve the merge conflicts here?

**Comment by jme783** [comment]
Thanks! Merging

**Pull request merged**